locales/templates/app.pot

```diff
@@ -391,6 +395,14 @@ msgctxt "onboarding_turbo_mode_button_off"
 msgid "Turn Turbo Mode Off"
 msgstr ""
 
+msgctxt "button_keep_enabled"
+msgid "Keep enabled"
+msgstr ""
```

Comment:
Oh actually I almost didn't notice this. 
In some languages "enabled" will be written differently whether it's related to a feminine noun vs. a masculine noun (since it's an adjective).
So, we'll need to specify what the noun is here.